docs/install.md

```diff
@@ -105,6 +105,12 @@ The compatible MMTracking, MMCV, and MMDetection versions are as below. Please i
     pip install -v -e .  # or "python setup.py develop"
     ```
 
+7. Install extra dependencies for VOT evaluation
+
+    ```shell
+    pip install vot-toolkit@git+ssh://git@github.com/votchallenge/vot-toolkit-python@0c61b32
```

Comment:
Let's simplify the command.
Please change all `pip install vot-toolkit@git+ssh://git@github.com/votchallenge/vot-toolkit-python@0c61b32` to `pip install git+https://github.com/votchallenge/toolkit.git`

And, add the command in the last of [A from-scratch setup script](https://github.com/open-mmlab/mmtracking/blob/master/docs/install.md#a-from-scratch-setup-script).